tests/ci/get_robot_token.py

```diff
@@ -66,10 +67,6 @@ def get_parameters_from_ssm(
 def get_best_robot_token(token_prefix_env_name="github_robot_token"):
     # Re-use already fetched token (same as in get_best_robot_token_original)
     # except here we assume it is always a string (since we use only one token and don't do token rotation)
-    global ROBOT_TOKEN
-    if ROBOT_TOKEN is not None:
-        return ROBOT_TOKEN
-    ROBOT_TOKEN = get_parameter_from_ssm(token_prefix_env_name)
     return ROBOT_TOKEN
```

Comment:
It looks like `ROBOT_TOKEN` is always going to be `None`, due assignment above:
```python
ROBOT_TOKEN = None  # type: Optional[Token]
```
https://github.com/Altinity/ClickHouse/pull/693/files#diff-51dab353e9c4821ee2c243ba8aaf19f39f3f5fdeba7657c76774d2ab82bd597fL60